cmd/massastation/main.go

```diff
@@ -56,23 +58,27 @@ func ParseFlags() api.StartServerFlags {
 }
 
 func main() {
-	logger.Logger = logger.NewLogger()
-	defer logger.Logger.Sync()
+	err := initialize.Logger()
+	if err != nil {
+		log.Fatalf("while initializing logger: %s", err.Error())
+	}
+
+	defer logger.Close()
 
 	flags := ParseFlags()
 	if flags.Version {
-		logger.Logger.Infof("Version:%s", config.Version)
+		logger.Infof("Version:%s", config.Version)
```

Comment:
I think we should use the regular log for this one : 
```
$ massastation --version
2023-07-07T17:22:21.790+0200	INFO	massastation/main.go:64	Version:v0.2.5

```

Reply:
what is the regular log?

Reply:
I simply printed it. I also fixed a linter warning.